authz/opa-bridge/inputs/create-table.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
is this a test file?

Reply:
mostly wondering because of the hardcoded user uuid

Reply:
Those are just dummies. the inputs folder is used by OPA VSCode plugins to allow autocompletion for inputs. You are supposed to fill it with examples